frontend/packages/console-shared/src/components/dropdown/ResourceDropdown.tsx

```diff
@@ -158,6 +180,7 @@ class ResourceDropdown extends React.Component<ResourceDropdownProps, State> {
     updateSelection: boolean,
   ) => {
     const unsortedList = { ...appendItems };
+    const resourceList = {};
```

Comment:
Hmm... is this needed anymore, you're setting it up front as props come in... or am I not following things?

Reply:
You are absolutely right, this isn't needed anymore after the last suggested change. My mistake, i have removed it